.github/workflows/ci.yml

```diff
@@ -83,14 +142,78 @@ jobs:
         run: echo "$HOME/.mix/escripts" >> $GITHUB_PATH
       - name: generate contracts
         run: make elixir-generate
-      - name: run credo
-        run: cd elixir && mix credo
-      - name: run dyalizer
-        run: cd elixir && mix dialyzer
       - name: Check for uncommited schema changes
         run: |
           git add -N elixir/
           git diff
           git diff --quiet
       - name: Git status
         run: git status
+
```

Comment:
are we checking or generating?

Reply:
we regenerate and check if there are some differences, in order to force the developer to generate and commit the changes, so the generated files will be always aligned with the protobuf definition

Reply:
should'nt split the check and the regeneration in two separate steps then?

Reply:
The checking is strictly dependent on the generation on the step before, and for the files to be present in the directory in order to git diffing, I think we can leave that as is